storage/DevicegraphImpl.cc

```diff
@@ -559,6 +583,8 @@ namespace storage
 
 		if (is_disk(device))
 		    out << ", color=\"#ff0000\", fillcolor=\"#ffaaaa\"";
+		else if (is_md(device))
+		    out << ", color=\"#aaaa00\", fillcolor=\"#ffffaa\"";
```

Comment:
I agree but on the other hand a change of the colours was never requested since the graphs exists (about 7 years). Since I don't have a style sheet language at hand I think this deserves a dedicated feature request.